imagemounter/volume_system.py

```diff
@@ -279,6 +305,11 @@ def _detect_mmls_volumes(self, vstype='detect'):
                 except Exception as e:
                     logger.exception("Failed executing mmls command")
                     raise SubsystemError(e)
+            # since mmls cannot determine the partition type for exFAT superfloppy formatting; check for this.
+            elif self._detect_exfat_superfloppy():
```

Comment:
I feel this method is in the wrong place. It should detect somewhere else, instead of randomly inside the various detect methods.